commons-webui-component/src/main/java/org/exoplatform/webui/form/UIFormRichtextInput.java

```diff
@@ -210,7 +210,6 @@ private String buildEditorLayout(WebuiRequestContext context) throws Exception {
             .append("CKEDITOR.plugins.addExternal('confirmBeforeReload','/commons-extension/eXoPlugins/confirmBeforeReload/','plugin.js');")
             .append("$('[name=\\'").append(name).append("\\']').ckeditor({")
             .append("customConfig: '/commons-extension/eXoConfig.js',")
-            .append("extraPlugins: 'confirmBeforeReload',")
```

Comment:
This plugin was introduced to show a popup when a user adds an input in TextArea and tries to change/reload the page in this commit https://github.com/exoplatform/commons/commit/320df4870aaded70c607ab3f124412f6579737cb .
Could you add it by default in eXoConfig.js

Reply:
done

Reply:
done